### PR TITLE
Uninstall OCS 4.5

### PR DESCRIPTION
**Pull request by annagitel**
this PR focuses on validating that all OCS resources are deleted from the OCP cluster. 
- test_uninstall - validate the OCS uninstall
- adding OCS uninstall automated function to the cluster teardown process
- changed storage classes removal condition (updated to 4.5)

Signed-off-by: annagitel <asandler@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/uninstall.py [hunk 1]
```diff
@@ -232,3 +234,100 @@ def uninstall_ocs():
         node_obj.add_label(resource_name=node, label=constants.TOPOLOGY_ROOK_LABEL + '-')
 
     log.info("OCS was removed successfully from cluster ")
+
+
+def uninstall_ocs45():
```
Comment (ebenahar) [inline ocs_ci/ocs/uninstall.py:239]: we should not have a function specific to a version. Please incorporate the implementation here in the existing uninstall function with conditions for the different versions

Comment (annagitel) [reply ocs_ci/ocs/uninstall.py:239]: dummy function just to test the new steps and flow (fully worked :) ) 
will work on merging the two funcs in the next days

Comment (ebenahar) [reply ocs_ci/ocs/uninstall.py:239]: thanks

**Comment by annagitel** [comment]
closing PR due to plan changes 
will follow up in a couple of separate PRS

**Pull request closed without merging**